src/test/transaction_tests.cpp

```diff
@@ -342,11 +342,26 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     string reason;
     BOOST_CHECK(IsStandardTx(t, reason));
 
-    t.vout[0].nValue = 501; // dust
+    // Check dust with default relay fee:
+    CAmount nDustThreshold = 182 * minRelayTxFee.GetFeePerK()/1000 * 3 ;
```

Comment:
`GetDustThreshold` is called via `IsStandardTx` so for the tests it's better not used here, imo. But thanks for the white space nit.